src/backend/funmooc/transfer/management/commands/import_fixtures.py

```diff
@@ -0,0 +1,75 @@
+"""Import fixtures management command."""
+import json
+import time
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+import gspread
+from oauth2client.service_account import ServiceAccountCredentials
+
+from ...import_scripts.categories import import_categories
+from ...import_scripts.courses import import_courses
+from ...import_scripts.organizations import import_organizations
+from ...import_scripts.persons import import_persons
+
+# If modifying these scopes, delete the file token.pickle.
+SCOPES = ["https://spreadsheets.google.com/feeds"]
+
+
+class Command(BaseCommand):
+    """Import pages from a Google Sheet."""
+
+    help = __doc__
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "google_sheet_id",
+            nargs="?",
+            default=settings.GOOGLE_SHEET_ID,
+            help="Google Sheet ID from which to import fixtures",
+        )
+
+    def import_pages(self, sheet, method, message):
+        """
+        Wrapper that adds displaying message and timing information when calling a given method.
+        """
+        self.stdout.write(message + " ", ending="")
```

Comment:
I would use the logging module instead. It's up the logger configuration to decide if it should be streamed to stdout given a log level, handler, etc.

Reply:
I don't agree on this one. This script is only to be launched manually and the printout is not interesting to be logged. I follow the same pattern as the migrate command for example: https://github.com/django/django/blob/master/django/core/management/commands/migrate.py#L162

Reply:
I don't agree with Django's streams management in this case. IMO all outputs should use the logging layer.

Reply:
We need to discuss this IRL because I don't see how this dot-dot progression can be handled by logging. What Django's `test` or `migrate` management commands are outputing is captured by your CI and you can access it where it matters.